src/Io/MultipartParser.php

```diff
@@ -67,18 +81,29 @@ private function __construct(ServerRequestInterface $request)
         if ($var !== false) {
             $this->maxInputNestingLevel = (int)$var;
         }
+
+        $this->uploadMaxFilesize = $uploadMaxFilesize === null ? $this->iniUploadMaxFilesize() : (int)$uploadMaxFilesize;
```

Comment:
Does is make sense to also allow a string similar to the ini setting, like `"8M"`? The conversion logic is already [there](https://github.com/reactphp/http/pull/274/files#diff-e583461d932d2fbc3b560b92c2eeb8adR333).

Reply:
Good point! I'm not opposed, but would rather look into this in a follow-up PR because this also affects the `RequestBodyBufferMiddleware` :+1:

Reply:
Opened #275 for that 👍